salvo/tools/typecheck.sh

```diff
@@ -26,4 +26,4 @@ fi
 
 echo $PWD
 
-${PYTYPE} src -P bazel-bin:.
+${PYTYPE} src -P bazel-bin:. --disable=pyi-error
```

Comment:
Can we add a comment explaining what are we disabling here and why?

Reply:
It will failed with below error without disabling this:
```
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/type_pb2.py:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/empty_pb2.py:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/descriptor_pb2.py:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/wrappers_pb2.py:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/api_pb2.py:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: Elapsed time: 33.397s, Critical Path: 11.85s
INFO: 254 processes: 56 internal, 198 processwrapper-sandbox.
INFO: Build completed successfully, 254 total actions
++ which pytype
+ PYTYPE=
+ true
+ '[' -z '' -a -f /root/.local/bin/pytype ']'
+ PYTYPE=/root/.local/bin/pytype
+ '[' -z /root/.local/bin/pytype ']'
+ echo /envoy-perf/salvo
/envoy-perf/salvo
+ /root/.local/bin/pytype src -P bazel-bin:.
Computing dependencies
Analyzing 35 sources with 5 local dependencies
ninja: Entering directory `.pytype'
[7/40] check src.lib.source_tree
FAILED: /envoy-perf/salvo/.pytype/pyi/src/lib/source_tree.pyi
/usr/bin/python3.10 -m pytype.single --imports_info /envoy-perf/salvo/.pytype/imports/src.lib.source_tree.imports --module-name src.lib.source_tree --platform linux -V 3.10 -o /envoy-perf/salvo/.pytype/pyi/src/lib/source_tree.pyi --analyze-annotated --enable-nested-classes --nofail --quick /envoy-perf/salvo/src/lib/source_tree.py
File "/envoy-perf/salvo/src/lib/source_tree.py", line 12, in <module>: Couldn't import pyi for 'api.source_pb2' [pyi-error]
  Can't find pyi for 'global___FieldDescriptorProto.Label', referenced from 'google.protobuf.descriptor_pb2'

For more details, see https://google.github.io/pytype/errors.html#pyi-error
ninja: build stopped: subcommand failed.
Leaving directory '.pytype'
```

It appears to be a bug: https://github.com/google/pytype/issues/764. So I added this workaround here, I will add a comment.